packages/slider/src/slider.ts

```diff
@@ -74,6 +75,12 @@ export class Slider extends Focusable {
     /* Ensure that a '' value for `variant` removes the attribute instead of a blank value */
     private _variant = '';
 
+    @property({ attribute: false })
+    public getAriaValueText: (value: number) => string = (value) => `${value}`;
+
+    @property({ reflect: true, attribute: 'aria-valuetext' })
```

Comment:
I'm fairly certain that this doesn't "reflect" to the parent. You referenced the React version, so I was looking there and `aria-valuetext` is only applied to the `input` there.

Reply:
The reflect on this property does get applied to the host. 
![image](https://user-images.githubusercontent.com/332843/70945920-09d67480-200b-11ea-9666-722e566fef39.png)

So I've not actually been following the aria stuff too much yet wrt WCs. Do we need the aria attributes just on the active focused input, or also on the slider itself?

Reply:
It's likely that @jnurthen has some added insight here as I've found even "clear" documentation leaves things to be desired. However, reviewing the MDN page for this attribute, there are a couple of things that look likely to me:

1. We shouldn't append it to any element unless we are actually representing a non-simple number: 
> Authors SHOULD only set the `aria-valuetext` attribute when the rendered value cannot be accurately represented as a number.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuetext_attribute

2. We shouldn't be applying `aria-valuetext` multiple times for one "widget': https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role#Examples

3. We should be applying `aria-valuetext` on the same element as `aria-valuemin`, `aria-valuemax`, and `aria-valuenow`. 

Maybe James can speak to a more comprehensive internal doc for slider based accessibility as similar docs for the Toast, etc patterns found [here](https://github.com/adobe/react-spectrum-v3/tree/master/specs/accessibility) have been very useful.

Reply:
Does the `Authors SHOULD only set the aria-valuetext attribute when the rendered value cannot be accurately represented as a number.` actually exclude adding it even if the rendered value is a number? It would certainly keep the code paths simpler to just always render `aria-valuetext` and just put the number in it.

Reply:
Updated in b49eddf, also added `aria-valuenow` to the input for good measure.

Reply:
@Westbrook I agree with your comments above. Unfortunately, we don't have a spec for slider for v3 yet. The best I can do is point you to the Accessibility section of the v2 documentation at https://react-spectrum.corp.adobe.com/components/Slider